### DISCUSSION TITLE
How do I configure client certificate in MySql?

### DISCUSSION THREAD

**Discussion by GorgWorgington** [diesel-rs/diesel#3624] [Q&A] [answered]
Configuring `ssl_cert` and `ssl_key` seems to be possible with the connection string in Postgres according to #2665 but it doesn't seem to be supported for MySql as far as I can tell.

**Comment by weiznich** [comment, marked as answer]
We accept `ssl_cert` and `ssl_mode` parameters for mysql connections as well: https://docs.diesel.rs/2.0.x/diesel/mysql/struct.MysqlConnection.html#method.establish
By looking at the documentation I notice that it might be useful to show that as well in the URL example on top of the `MysqlConnection` documentation page. Mind submitting a PR for that?
`ssl_key` is not supported yet by diesel, but I would accept a PR that adds support for that as well.

**Reply by GorgWorgington** [reply]
If I have some extra time I'll take a stab at PRs for those things. Thanks for responding so quickly.
